python/GafferOFXUITest/__init__.py

```
@@ -0,0 +1,41 @@
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+from .DocumentationTest import DocumentationTest
+from .NodeUITest import NodeUITest
+
+if __name__ == "__main__":
+	unittest.main()
```

Ruff (0.8.2)

Undefined name `unittest`

(F821)